test/outdir-convert/valid.test.mjs

```diff
@@ -3,44 +3,120 @@ import * as assert from 'uvu/assert';
 import { extname, basename, resolve } from 'path';
```

Comment:
There are more changes to the test suite than I wanted to make initially for this feature add. Considering there are now two kinds of things being formatted and tested (.mjs and .mjs.map), I decided to make organizational changes to more clearly express what was being tested.

Reply:
First take here is spitting the concerns into different files for the api versus cli might make sense.

I'm going to grab this branch locally and take a look at what this looks like.